From: Ben Hutchings <ben@decadent.org.uk>
Subject: wireless: orinoco_usb is BROKEN
Date: Sun, 31 Mar 2013 07:31:24 +0100

orinoco_usb plays some funny games with completions, which might just
about work in mainline but don't work with the type changes in RT.
I'm not going to fix this crap.

---
--- a/drivers/net/wireless/orinoco/Kconfig
+++ b/drivers/net/wireless/orinoco/Kconfig
@@ -135,7 +135,7 @@ config PCMCIA_SPECTRUM
 
 config ORINOCO_USB
 	tristate "Agere Orinoco USB support"
-	depends on USB && HERMES
+	depends on USB && HERMES && BROKEN
 	select FW_LOADER
 	---help---
 	  This driver is for USB versions of the Agere Orinoco card.
